helper/keysutil/policy_test.go

```diff
@@ -245,15 +246,13 @@ func testArchivingCommon(t *testing.T, lm *LockManager) {
 		KeyType: KeyType_AES256_GCM96,
 		Name:    "test",
 	})
-	if lock != nil {
-		defer lock.RUnlock()
-	}
 	if err != nil {
 		t.Fatal(err)
 	}
-	if p == nil {
-		t.Fatal("nil policy")
+	if p == nil || lock == nil {
+		t.Fatal("nil policy or lock")
 	}
+	lock.RUnlock()
```

Comment:
Missing defer in here?

Reply:
This should be unlocked right away. In theory in the rest of the test we should be grabbing a write lock on the policy, but since it's test-scoped and not in parallel we don't.